fdbserver/tester.actor.cpp

```diff
@@ -1146,6 +1146,9 @@ ACTOR Future<bool> runTest(Database cx,
 std::map<std::string, std::function<void(const std::string&)>> testSpecGlobalKeys = {
 	// These are read by SimulatedCluster and used before testers exist.  Thus, they must
 	// be recognized and accepted, but there's no point in placing them into a testSpec.
+	// testClass and testPriority are only used for TestHarness, we'll ignore those here
+	{ "testClass", [](std::string const&) {} },
```

Comment:
it wasn't intended for that. For example all authz tests require multi-tenancy to be enabled. But their test class shouldn't be multi-tenancy related since they test something else. So I think it would be relatively difficult to use this for filtering